Sources/SmileID/Classes/Networking/ServiceRunnable.swift

```diff
@@ -219,18 +264,29 @@ extension ServiceRunnable {
     private func createRestRequest(
         path: PathType,
         method: RestMethod,
-        headers: [HTTPHeader]? = nil,
+        headers: [HTTPHeader],
         queryParameters: [HTTPQueryParameters]? = nil
     ) throws -> RestRequest {
         let path = String(describing: path)
         guard let url = baseURL?.appendingPathComponent(path) else {
             throw URLError(.badURL)
         }
 
+        var signedHeaders = headers
+        do {
+            if let header = headers.first(where: { $0.name == "SmileID-Request-Timestamp" }) {
+                let requestMac = try SmileIDCryptoManager.shared.sign(
+                    timestamp: header.value,
+                    headers: headers.toDictionary()
+                )
+                signedHeaders += [.requestMac(value: requestMac)]
+            }
+        } catch { /* in case we can't add the security info the backend will deal with the enrollment */ }
```

Comment:
do-catch seems redundant since we are not catching any errors.

Reply:
Resolved in [f1c0a73](https://github.com/smileidentity/ios/pull/286/commits/f1c0a7342480f07515b317fc20349c7a7ddf0bd5)